src/components/Sections/Home/LatestDailyLesson.js

```diff
@@ -20,8 +20,9 @@ const getRandomImage = () => {
 };
 
 const LatestDailyLesson = ({ collection, t }) => {
-  // eslint-disable-next-line no-unused-vars
-  const [imageSrc, _] = useState(getRandomImage());
+  const [imageSrc, setImage] = useState();
+
+  useEffect(() => setImage(getRandomImage()), []);
```

Comment:
I don't understand why this is better then initial value in useState()

Reply:
adding useEffect gets rid of a warning:
Warning: Prop `src` did not match. Server: "null" Client: "https://kabbalahmedia.info/imaginary/thumbnail?url=https%3A%2F%2Fkabbalahmedia.info%2Fassets%2Flessons%2Flatest_lesson_6.jpg&width=512&height=288&nocrop=false&stripmeta=true"
    in img (created by Image)
    in Image (at LatestDailyLesson.js:30)
    in a (created by LinkAnchor)
    in LinkAnchor (created by Context.Consumer)
    in Link (at MultiLanguageLinkCreator.js:32)
    in MultiLanguageLinkHOC (at LatestDailyLesson.js:29)
    in div (at LatestDailyLesson.js:28)
    in LatestDailyLesson (created by NamespacesConsumerComponent)
    in NamespacesConsumerComponent (created by WithMergedOptions)
    in WithMergedOptions (created by Context.Consumer)
    in WithMergedOptions(NamespacesConsumerComponent) (created by LoadNamespace)
    in LoadNamespace (created by WithMergedOptions)
    in WithMergedOptions (created by Context.Consumer)
    in LoadNamespace(LatestDailyLesson) (at HomePage.js:171)

Reply:
Our target is not to delete messages, but understand what they are about, what clue they try to give us